bin/spawn_devcontainer.sh

```diff
@@ -95,21 +95,12 @@ SETUP_ENVIRONMENT_DIR="${REMOTE_HOME}/$setup_environment_dir"
 devcontainer up ${remove_existing_container} \
 	--dotfiles-repository "${setup_environment_repo}" \
 	--dotfiles-target-path "${SETUP_ENVIRONMENT_DIR}" \
+  --dotfiles-install-command "${setup_environment_install_command}" \
 	--workspace-folder "${workspace}" \
 	--update-remote-user-uid-default off
-# Only the clonation is needed, the installation does nothing. This is done in the next commands (with ROOT user)
-# --dotfiles-install-command "${nvim_dotfiles_install}" \
-
-# Setting Up Devcontainer
-# Installing dependencies as root (nvim package)
-DEVCONTAINER_ROOT_CONFIG=.devcontainer/autogenerated-devcontainer-root.json
-
-jq -n --arg workspaceFolder "$WORKSPACE_FOLDER" '{ "remoteUser": "root", "workspaceFolder": $workspaceFolder, "remoteEnv": { "HOME": "" }}' >${DEVCONTAINER_ROOT_CONFIG}
-# jq -n '{ "remoteUser": "root", "remoteEnv": { "HOME": "" }}' >${DEVCONTAINER_ROOT_CONFIG}
-# devcontainer up --dotfiles-repository clones the dotfiles by default in this path /home/${REMOTE_USER}/dotfiles
-devcontainer exec --override-config ${DEVCONTAINER_ROOT_CONFIG} --workspace-folder "${workspace}" sh -c "cd ${SETUP_ENVIRONMENT_DIR} && ${setup_environment_install_command}"
-rm "${DEVCONTAINER_ROOT_CONFIG}"
```

Comment:
After some extensive testing it seems that `devcontainer-cli` no longer allows for an actual command to be set. It takes a file name and that is it. The help kinda indicates this, but it isn't clear:
```
--dotfiles-install-command        The command to run after cloning the dotfiles repository. Defaults to run the first                                                                                                                                               
                                                 file of `install.sh`, `install`, `bootstrap.sh`, `bootstrap`, `setup.sh` and `setup`                                                                                                                                             
                                                 found in the dotfiles repository`s root folder.                            [string]                                                                                                                                             
```

Reply:
Sorry I posted this two days ago, but it marked it as a review comment so it never showed, since I didn't say complete review. Anyway:

I looked at the diff and that command was never used in the original shell script. See line 101. So in fact, that would be a bug with the code you originally had and it explains issues I was having before. I would only use the builtin dotfiles commands and it was never working correctly.